MIP14/MIP14c2-Subproposals/MIP14c2-SP5.md

```diff
@@ -9,9 +9,10 @@ Contributors:
 Tags: tech-001, core-unit,
 Status: Formal Submission
 Date Proposed: 2023-02-01
-Date ratified: -
+Date ratified: N/A
 Forum URL: https://forum.makerdao.com/t/mip14c2-sp5-requesting-protocol-dai-transfer-to-fund-techops-cu-for-march-2023/19624
-Ratification Poll URL: 
+Ratification Poll URL: https://vote.makerdao.com/polling/Qmdzxxw4
+Extra: This proposal has been marked as obsolete as per the changes introduced in [MIP0](https://mips.makerdao.com/mips/details/MIP0) via [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1).
```

Comment:
For consistency, write: Extra: This MIP has been made obsolete by the passage of [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1)

This was what we wrote in all the MIPs we rendered obsolete.